### PR TITLE
Updated sample for 204

### PR DESCRIPTION
**Pull request by ooeygui**
Revised to use git submodule

### CODE REVIEW

File: Lesson_204V2/FullSolution/MainPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Devices.Adc;
+using Windows.Media.SpeechSynthesis;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+using Microsoft.IoT.AdcMcp3008;
+// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
```
Comment (mwmckee) [inline Lesson_204V2/FullSolution/MainPage.xaml.cs]: comment no longer needed

**Comment by turkycat** [comment]
This PR does not contain edits to Lesson_203, however... the StartSolution does not compile. It is missing `CalibrationData` member in `BMP280.cs`.

Please add these lines to `BMP280.cs`, as found in FullSolution of the same lesson:

```C#
        //Create new calibration data for the sensor
        BMP280_CalibrationData CalibrationData;
```

**Comment by turkycat** [comment]
Why do we have both "Lesson_204" and "Lesson_204V2" contained in this repo? Neither of the StartSolutions contained in both "Lesson_204" and "Lesson_204V2" compile, are both missing several members. Also, If Build > Rebuild All is selected in "Lesson_204V2" is selected, rebuild fails due to skipping the project. Error is:

```
1>------ Skipped Rebuild All: Project: Lesson_204V2, Configuration: Debug x86 ------
1>Project not selected to build for this solution configuration 
========== Rebuild All: 0 succeeded, 0 failed, 1 skipped ==========
```

If right click on the project > rebuild, still fails for missing members in `MainPage.xaml.cs`

**Comment by turkycat** [comment]
Naive question: Why is there no "Lesson_202"?

**Comment by turkycat** [comment]
All other solutions build successfully from a new clone. These include:

Blinky, both
Lesson_201, both
Lesson_203, FullSolution only
Lesson_204, FullSolution only
Lesson_204V2, FullSolution only
Lesson_205, both

File: Lesson_204V2/StartSolution/MainPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Devices.Adc;
+using Windows.Media.SpeechSynthesis;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+using Microsoft.IoT.AdcMcp3008;
+// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
```
Comment (turkycat) [inline Lesson_204V2/StartSolution/MainPage.xaml.cs]: comment no longer needed here as well, as noticed by @mwmckee in FullSolution (this is StartSolution)

File: Lesson_204V2/StartSolution/MainPage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                Windows.UI.Core.CoreDispatcherPriority.High,
+                async () =>
+                {
+                    // <Add speech code here>
+                }
+            );
+        }
+
+        // This will put our pin on the world map of makers
+        // Go to ENTER FINAL LINK HERE to view your pin
```
Comment (turkycat) [inline Lesson_204V2/StartSolution/MainPage.xaml.cs]: replace "ENTER FINAL LINK HERE" with the link

File: Lesson_204V2/FullSolution/MainPage.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+                    // start this audio stream playing
+                    media.AutoPlay = true;
+                    media.SetSource(synthesisStream, synthesisStream.ContentType);
+                    media.Play();
+                }
+            );
+        }
+
+        // This will put our pin on the world map of makers
+        // Go to ENTER FINAL LINK HERE to view your pin
```
Comment (apdutta) [inline Lesson_204V2/FullSolution/MainPage.xaml.cs]: "ENTER FINAL LINK HERE" should be http://ms-iot.github.io/content/en-US/win10/samples/BrightOrNot.htm

File: Lesson_204V2/FullSolution/MainPage.xaml.cs (hunk 1 above)
Comment (ooeygui) [inline Lesson_204V2/FullSolution/MainPage.xaml.cs]: :heavy_check_mark:

File: Lesson_204V2/StartSolution/MainPage.xaml.cs (hunk 2 above)
Comment (ooeygui) [inline Lesson_204V2/StartSolution/MainPage.xaml.cs]: :heavy_check_mark:

File: Lesson_204V2/StartSolution/MainPage.xaml.cs (hunk 3 above)
Comment (ooeygui) [inline Lesson_204V2/StartSolution/MainPage.xaml.cs]: :heavy_check_mark:

File: Lesson_204V2/FullSolution/MainPage.xaml.cs (hunk 4 above)
Comment (ooeygui) [inline Lesson_204V2/FullSolution/MainPage.xaml.cs]: :heavy_check_mark:

**Comment by ooeygui** [comment]
202 was cut from the initial kit. It was supposed to be added in a quick follow up.
I don't know why V2 is a separate project. I asked that myself, but everyone is out.

**Comment by ooeygui** [comment]
My understanding is that the start solutions won't compile unless you go through the walkthrough. I'm not sure how much I want to fix those up at the moment. So I'm going to leave them as is

**Comment by apdutta** [comment]
:+1:

**Pull request merged**